app/src/main/res/layout/audio_fragment.xml

```diff
@@ -35,6 +35,7 @@
       android:contentDescription="@{viewModel.playStatusLiveData == UiAudioPlayStatus.PLAYING ? @string/audio_pause_description : @string/audio_play_description}"
       android:onClick="@{(v) -> viewModel.togglePlayPause(viewModel.playStatusLiveData)}"
       android:padding="12dp"
+      app:tint="@color/component_color_audio_fragment_icon_background_color"
```

Comment:
This is incorrect as it is not a `background_color`. Please create a new value which makes sense here.